<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #e6ebdc;
}

QLineEdit, QTextEdit {
	background-color: #a9af9e;
	color: #000000;
	border: none;
	border-radius: 12px;
	padding: 6px;
	font-size: 15px;
}

QPushButton {
	background-color: #4A7C59;
	color: white;
	border: none;
	border-radius: 12px;
	padding: 6px 12px;
}

QPushButton:hover {
	background-color: #5A9F73;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="pt_nid_e_lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pt_male_e_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>245</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_pt_e_Button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>299</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pt_birthday_e_comboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style for the QComboBox */
QComboBox {
	border-radius: 10px;
	padding-left: 10px;
}

/* style for drop down area */
QComboBox::drop-down {
	border: 0px;
}

/* style for drop down arrow */
QComboBox::down-arrow {
	image: url(D:\Python/Projects/GitHub_PR/Hospital Management/down-arrow-8-128);
	width: 12 px;
	height: 12px;
	margin-right: 10px;
}

/* style for list menu */
QComboBox QListView {
font-size: 12px;
border: 1px solid #000000;
padding: 5px;
background-color: #a9af9e;
outline: 0px;
}

QComboBox {
	background-color: #a9af9e;
    font-size: 15px;
    font-family: &quot;Arad Medium&quot;;
}

QComboBox QAbstractItemView {
    background-color: #a9af9e;
    color: black;
    selection-background-color: #F3E9E2;
    selection-color: black;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pt_ln_e_lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pt_gender_e_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pt_phone_e_lineEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pt_birthdate_e_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>138</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Birthdate:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="pt_brithmonth_e_comboBox">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>140</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style for the QComboBox */
QComboBox {
	border-radius: 10px;
	padding-left: 10px;
}

/* style for drop down area */
QComboBox::drop-down {
	border: 0px;
}

/* style for drop down arrow */
QComboBox::down-arrow {
	image: url(D:\Python/Projects/GitHub_PR/Hospital Management/down-arrow-8-128);
	width: 12 px;
	height: 12px;
	margin-right: 10px;
}

/* style for list menu */
QComboBox QListView {
font-size: 12px;
border: 1px solid #000000;
padding: 5px;
background-color: #a9af9e;
outline: 0px;
}

QComboBox {
	background-color: #a9af9e;
    font-size: 15px;
    font-family: &quot;Arad Medium&quot;;
}

QComboBox QAbstractItemView {
    background-color: #a9af9e;
    color: black;
    selection-background-color: #F3E9E2;
    selection-color: black;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="pt_detail_e_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>If You Have Or Had Any Sickness                              Write That Down Here</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pt_phone_e_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Phone Number:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pt_birthyear_e_comboBox">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>140</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* style for the QComboBox */
QComboBox {
	border-radius: 10px;
	padding-left: 10px;
}

/* style for drop down area */
QComboBox::drop-down {
	border: 0px;
}

/* style for drop down arrow */
QComboBox::down-arrow {
	image:url(D:\Python/Projects/GitHub_PR/Hospital Management/down-arrow-8-128);
	width: 12 px;
	height: 12px;
	margin-right: 10px;
}

/* style for list menu */
QComboBox QListView {
font-size: 12px;
border: 1px solid #000000;
padding: 5px;
background-color: #a9af9e;
outline: 0px;
}

QComboBox {
	background-color: #a9af9e;
    font-size: 15px;
    font-family: &quot;Arad Medium&quot;;
}

QComboBox QAbstractItemView {
    background-color: #a9af9e;
    color: black;
    selection-background-color: #F3E9E2;
    selection-color: black;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pt_detail_e_textEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="top_inf_p_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Change What You want!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pt_fn_e_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pt_fn_e_lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pt_female_e_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>245</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pt_nid_e_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>National ID:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pt_ln_e_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arad Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="pt_e_female_radioButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>247</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="pt_e_male_radioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>247</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
